src/layer/photos/index.md

```diff
@@ -1,27 +1,33 @@
 # Capturing Photos
+[[toc]]
 
 When surveying, you might want to take a photo from your camera or attach an existing photo from the device gallery to your survey feature.
 
-Here, you will find the overview of the basic functionality related to capturing photos, such as:
-[[toc]]
+To capture and save photos using the <MobileAppNameShort />, the survey layer needs to have a field configured with the [attachment widget](../form-widgets/#attachment). In the <MobileAppNameShort />, this field will provide two options:
+- **Take a picture** to use your camera app for taking a photos 
+- **From gallery** to attach an existing photo from your device
+
+![Mergin Maps mobile app photo widget](../form-widgets/mobile-form-attachment-photo.jpg "Mergin Maps mobile app photo widget")
 
 :::tip
-Do you need to attach multiple pictures to one feature? [How to attach multiple photos to features](./attach-multiple-photos-to-features/) will guide you through the setup.
+Do you need to attach multiple pictures to one feature? [How to attach multiple photos to features](../attach-multiple-photos-to-features/) will guide you through the setup.
 :::
 
 ::: warning
 Keep in mind that synchronising photos during the field survey can use up a lot of mobile data - depending on the amount of pictures taken and their size.
 
-To reduce the data usage, you may consider using [selective synchronisation](../manage/selective_sync/), [resizing pictures automatically](#resizing-pictures-automatically) or following the [offline field survey workflow](../field/offline-use/#offline-field-survey-workflow). 
+To reduce the data usage, you may consider using [selective synchronisation](../../manage/selective_sync/), [resizing pictures automatically](#resizing-pictures-automatically) or following the [offline field survey workflow](../../field/offline-use/#offline-field-survey-workflow). 
 :::
 
-## Attachment widget in QGIS
+## Photo attachment widget in QGIS
 
-:::tip
+:::tip Example project available
 Clone <MerginMapsProject id="documentation/form_setup" /> to follow this example!
 :::
 
-To set up a photo widget:
+In QGIS, the [attachment widget](../form-widgets/#attachment) is used to set up a field to capture photos.
+
+To set up the attachment widget:
 1. Right-click on a layer, select **Properties** and go to the **Attributes form** tab.
 2. In the list of **Available Widgets** select the text field you want to work with.
 3. In the **Widget Display** tab:
```

Comment:
The `Absolute Path` option is in my opinion unusable altogether for working in a team. If used, people won't be able to see each others photos because paths differ from system to system. My suggestion would be to remove the three lines where we say "Store path as absolute path" from the table below and instead add a red warning saying that Absolute path option should not be used for collaborative work as it stores the whole path. It would not work even for a single person trying to set the project up between their mobile app and desktop.

In fact, we even report it as a warning when someone uses the absolute path option, see:
<img width="386" alt="image" src="https://github.com/user-attachments/assets/89f84419-6dde-45e2-923e-90f59cacaef6">

Reply:
![image](https://github.com/user-attachments/assets/aa347a13-956d-40a7-b4d0-8da038e3a04f)